upk2esphome/parts/remote.py

```diff
@@ -25,7 +28,11 @@ def generate(yr: YamlResult, config: ConfigData, opts: Opts):
         yr.log(f"Remote receiver: pin P{ir_pin}")
         receiver = {
             "pin": f"P{ir_pin}",
+            "id": f"P{ir_pin}"
```

Comment:
Probably don't need this, no config has both `ir` and `infrr`, it's always one or the other.

Reply:
alright, was wondering about that. will go back to a previous version (in between commits) :)